voicevox_engine/downloadable_library.py

```diff
@@ -0,0 +1,54 @@
+import json
+import zipfile
+from io import BytesIO
+from pathlib import Path
+from typing import List
+
+import requests
+from fastapi import HTTPException
+
+from voicevox_engine.model import DownloadableLibrary
+
+__all__ = ["LibraryManager"]
+
+INFO_FILE = "library_info.json"
```

Comment:
まあ問題ないとは思いつつも、わかりやすいように複数形にしときますか
```suggestion
INFO_FILE = "libraries_info.json"
```

Reply:
これ、`libraries/[uuid]/library_info.json`に来るはずなので単数形の方が正しいかも？

Reply:
ああ、本当ですね！
であればlibraryは要らなそうです。他のに合わせてmetas.jsonにするか、library_manifest.jsonにするのはどうでしょう。

Reply:
この辺の仕様は定まってなかったので、議論していきたいかもですね！
プロジェクト化しますか。

Reply:
metas.jsonにしました。